sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasOptionsDto.java

```diff
@@ -31,7 +32,8 @@
 	public static final String WITH_SAMPLES = "withSamples";
 	public static final String WITH_EVENT_PARTICIPANTS = "withEventParticipants";
 
-	private ServerAccessDataReferenceDto organization;
+	// Fixme this should be renamed but it has strange side effects with the UI
+	private SormasServerDescriptor organization;
```

Comment:
targetServer? Or keep organization and make sure to have it consistent in SormasToSormasShareInfo + SormasToSormasSHareInfoDto

Reply:
OK I will keep it for now. I filed #6101 to fix this after the PR.